kits/exam-question-paper-generator/prompts/exam-question-paper-generator_llmnode-644_user_1.md

```
@@ -0,0 +1,22 @@
+Generate a complete exam question paper with the following details:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)